app/dashboard/models.py

```diff
@@ -1065,6 +1078,67 @@ def stats(self):
             (success_rate, 'Success Rate'),
         ]
 
+    @property
+    def get_quarterly_stats(self):
+        user_active_in_last_quarter = False
+        last_quarter = datetime.now() - timedelta(days=90)
+        bounties = self.bounties.filter(modified_on__gte=last_quarter)
+        fulfilled_bounties = [
+            bounty for bounty in bounties if bounty.is_hunter(self.handle) and bounty.status == 'done'
+        ]
+        fulfilled_bounties_count = len(fulfilled_bounties)
+        total_earned_eth = sum([
+            bounty.value_in_eth if bounty.value_in_eth else 0
+            for bounty in fulfilled_bounties
+        ])
+        total_earned_eth /= 10**18
+        total_earned_usd = sum([
+            bounty.value_in_usdt if bounty.value_in_usdt else 0
+            for bounty in fulfilled_bounties
+        ])
+
+        num_completed_bounties = bounties.filter(idx_status__in=['done']).count()
+        terminal_state_bounties = bounties.filter(idx_status__in=['done', 'expired', 'cancelled']).count()
```

Comment:
Maybe we should define what `terminal_state` statuses are, so we can use the filter anywhere and know it matches this.